tests/unit/data_finder/test_get_start_end_year.py

```diff
@@ -34,6 +36,7 @@ def test_get_start_end_year(case):
 def test_read_time_from_cube():
     """Try to get time from cube if no date in filename"""
     temp_file = tempfile.NamedTemporaryFile(suffix='.nc')
+    temp_file.name = re.sub(r'\d', 'v', temp_file.name)
```

Comment:
```suggestion
def test_read_time_from_cube(tmp_path):
    """Try to get time from cube if no date in filename"""
    temp_file = str(tmp_path / 'test.nc')
```
I would recommend using the [`tmp_path`](https://docs.pytest.org/en/latest/tmpdir.html) fixture instead, it's much easier to understand

Reply:
absolutely! good call